packages/auth-construct/API.md

```diff
@@ -42,14 +42,10 @@ export type AuthProps = {
 };
 
 // @public
-export type EmailLogin = true | EmailLoginSettings;
+export type EmailLogin = true | EmailLoginSettings | Record<string, never>;

```

Comment:
What does `Record<string, never>` mean in this context ? Why would `EmailLogin` loosely typed?

Reply:
It represents the empty object type, which allowed us to retain empty object support; but I don't need it anymore. I needed it at one point because I had made one of the properties of EmailLoginSettings required.